crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -539,8 +622,26 @@ fn point_light(
     color = diffuse + specular_light;
 #endif  // STANDARD_MATERIAL_CLEARCOAT
 
+    var cookie_sample = 1f;
+
+#ifdef CLUSTERED_DECALS_ARE_USABLE
+    if enable_cookie && (*light).decal_index != 0xFFFFFFFFu {
+        let relative_position = (view_bindings::clustered_decals.decals[(*light).decal_index].local_from_world * vec4(P, 1.0)).xyz;
+        let cubemap_type = view_bindings::clustered_decals.decals[(*light).decal_index].tag;
+        let decal_uv = cubemap_uv(relative_position, cubemap_type);
+        let image_index = view_bindings::clustered_decals.decals[(*light).decal_index].image_index;
+
+        cookie_sample = textureSampleLevel(
```

Comment:
i used both textureSample and textureSampleLevel(0) in different places, i've switched them all to textureSampleLevel(0).

i think textureSample would be ok, we could use mips if available, but i understand unreal uses mips to blend from harder to softer shadows over distance, if we do something similar we would want to choose the level explicitly here.

Reply:
I agree with using `textureSampleLevel(0)` for now, though we probably will want to select a mip level at some point.